imosToolbox.m

```diff
@@ -48,7 +48,14 @@ function imosToolbox(auto, varargin)
 % POSSIBILITY OF SUCH DAMAGE.
 %
 
+% Set current toolbox version
+toolboxVersion = ['2.5.28 - ' computer];
+
 if nargin == 0, auto = 'manual'; end
+if nargin == 1 && strcmpi(auto, 'version')
```

Comment:
Do you mind if I change this to simply:
```Matlab
if strcmpi(auto, 'version')
```
Since worst case 'auto' is being defined by the line just above.